packages/daemon/src/app/controller/job.ts

```diff
@@ -26,16 +61,11 @@ export class JobController extends BaseEventController {
     const pipeline = await this.pipelineService.getPipeline(pipelineId);
     if (pipeline) {
       const plugins = await this.pipelineService.fetchPlugins(pipeline);
-      const job = await this.pipelineService.createJob(pipelineId);
-      const logPath = join(constants.PIPCOOK_RUN, job.id, 'logs');
-      const stdoutFile = join(logPath, 'stdout.log');
-      const stderrFile = join(logPath, 'stderr.log');
-      await ensureDir(logPath);
-      await [
-        ensureFile(stdoutFile),
-        ensureFile(stderrFile)
-      ];
-      const tracer = await this.traceManager.create({ stdoutFile, stderrFile });
+      const realParam = this.createParams(pipeline);
```

Comment:
```suggestion
      const params = this.createParams(pipeline);
```

The params might be enough to represent here :)